components/style/servo/restyle_damage.rs

```diff
@@ -218,7 +218,8 @@ fn compute_damage(old: &ServoComputedValues, new: &ServoComputedValues) -> Servo
         get_inheritedbox.direction, get_inheritedbox.writing_mode,
         get_text.text_decoration_line, get_text.unicode_bidi,
         get_inheritedtable.empty_cells, get_inheritedtable.caption_side,
-        get_column.column_width, get_column.column_count
+        get_column.column_width, get_column.column_count,
+        get_inheritedsvg.fill, get_svg.cx, get_svg.cy, get_svg.r
```

Comment:
Ok, so you actually want a style struct field, but _not_ exposing it as a CSS property... So at least you need to use `internal="True"` on the property definition, and document that it's not web-exposed.